cypress/integration/v2/blocks/video-popup.spec.js

```diff
@@ -42,6 +46,73 @@ function switchDesign() {
 	] ) )
 }
 
+function styleTab( viewport, desktopOnly, registerBlockSnapshots ) {
+	cy.setupWP()
+	cy.newPage()
+	cy.addBlock( 'ugb/video-popup' ).as( 'videoPopupBlock' )
+	const videoPopupBlock = registerBlockSnapshots( 'videoPopupBlock' )
+	cy.openInspector( 'ugb/video-popup', 'Style' )
+
+	cy.setBlockAttribute( {
+		'videoLink': Cypress.env( 'DUMMY_VIDEO_URL' ),
```

Comment:
remove single quote

Reply:
there's an eslint error when i remove the single quotes here